go.mod

```diff
@@ -1,9 +1,9 @@
 module github.com/IBM/cloudant-go-sdk
 
-go 1.14
+go 1.15
```

Comment:
What is the exact meaning of this line?
https://golang.org/ref/mod#go-mod-file-go tells me it's the expected language version. https://stackoverflow.com/a/58205552/5323166 and https://www.reddit.com/r/golang/comments/b0b8k3/what_version_should_the_gomod_go_directive_be_set/?utm_source=share&utm_medium=web2x&context=3 say it's meant as the minimum accepted version which according to https://github.com/IBM/cloudant-go-sdk#prerequisites is 1.13 now.

Reply:
From: https://golang.org/ref/mod#go-mod-file-go 
> The language version determines which language features are available when compiling packages in the module.

I thought it is better to install only one version from go to the desktop. This minimum accepted version aspect is new for me. I am okay to set it to 1.13. Change in 4b17402